MoltenVK/MoltenVK/API/vk_mvk_moltenvk.h

```diff
@@ -795,6 +795,18 @@ VKAPI_ATTR void VKAPI_CALL vkGetVersionStringsMVK(
     char*                                       pVulkanVersionStringBuffer,
     uint32_t                                    vulkanVersionStringBufferLength);
 
+/**
+ * Sets the number of threads in a threadgroup for a compute kernel.
+ *
+ * This needs to be called if you are creating compute shader modules from MSL
+ * source code or MSL compiled code. Threadgroup size is determined automatically
+ * if you're using SPIR-V.
+ */
+VKAPI_ATTR void VKAPI_CALL vkSetThreadgroupSizeMVK(
+    VkShaderModule                              shaderModule,
```

Comment:
@BearishSun 

My apologies...I didn't notice this when I did the original review...

The MoltenVK API is for Vulkan developers. _Threadgroup_ is a Metal term. I've already merged this PR...so can you submit another PR that changes the name `vkSetThreadgroupSizeMVK` to `vkSetWorkgroupSizeMVK`...to make it more _Vulkan-ish_, please?

I know the MoltenVK API also references things like `MTLTexture`...but that is explicitly tied to a Metal artifact.